### PR TITLE
[ci skip]Update RELEASE_NOTES

### PR DESCRIPTION
**Pull request by chengstory**

### CODE REVIEW

File: docs/RELEASE_NOTES.md [hunk 1]
```diff
@@ -153,18 +153,26 @@ cocos new -l cpp|js|lua MyNewGame
...
 We are happy to announce the release of Cocos2d-x v3.10. Following are the highlighted features, improvements and API updates in this version. 
 
-1. UI System:
-    - Reimplemented Scale9Sprite and improve the scale9sprite performance and reduce memory consumption.
-    - Changed PageView to derived from ListView, PageView can be added any widget as child.
-    - Added three overflow type to new label: CLAMP，SHRINK，RESIZE_HEIGHT.
-2. JSModule:
-    - Improved JS bindings with more secured memory management, and enhance the stability of the engine, streamline all JSB code with the [GC coding style of SpiderMonkey](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/GC_Rooting_Guide).
+1. Provides a unified setup for both Cocos2d-x and Cocos.
+2. New __Cocos Launcher__ too! __Cocos Launcher__:
```
Comment (pandamicro) [inline docs/RELEASE_NOTES.md]: too! -> tool ?
and why repeat it twice?

File: docs/RELEASE_NOTES.md [hunk 2]
```diff
@@ -153,18 +153,26 @@ cocos new -l cpp|js|lua MyNewGame
...
+  * one step creates a new project by either using our precompiled libraries or
```
Comment (pandamicro) [inline docs/RELEASE_NOTES.md]: Uppercase first letter

One step to create a new project by either using our precompiled libraries or source code engine

File: docs/RELEASE_NOTES.md [hunk 3]
```diff
@@ -153,18 +153,26 @@ cocos new -l cpp|js|lua MyNewGame
...
+    source code.
```
Comment (pandamicro) [inline docs/RELEASE_NOTES.md]: No new line here

File: docs/RELEASE_NOTES.md [hunk 4]
```diff
@@ -153,18 +153,26 @@ cocos new -l cpp|js|lua MyNewGame
...
+  * easily integrate SDKBOX for 3rd party services. Easily add Facebook, in-app
```
Comment (pandamicro) [inline docs/RELEASE_NOTES.md]: Uppercase first letter

Easy integration with SDKBOX for 3rd party services: Facebook, IAP and many more.

File: docs/RELEASE_NOTES.md [hunk 5]
```diff
@@ -153,18 +153,26 @@ cocos new -l cpp|js|lua MyNewGame
...
+    purchase and many more.
+  * provide feedback and get support, directly within __Cocos Launcher__, making
+    it simple to contact the engine team for help.
+3. UI System:
+    - Reimplemented `Scale9Sprite` and improve the scale9sprite performance and reduce memory consumption.
+    - Changed `PageView` to derived from ListView, PageView can be added any widget as child.
+    - Added three overflow type to new label: **CLAMP**，**SHRINK**，**RESIZE_HEIGHT**.
+5. JSModule:
+    - Improved JS Bindings: follows [SpiderMonkey](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/GC_Rooting_Guide) GC best practices making it more robust.
```
Comment (pandamicro) [inline docs/RELEASE_NOTES.md]: Please modify it as Riq suggested

File: docs/RELEASE_NOTES.md [hunk 6]
```diff
@@ -153,18 +153,26 @@ cocos new -l cpp|js|lua MyNewGame
...
-2. JSModule:
-    - Improved JS bindings with more secured memory management, and enhance the stability of the engine, streamline all JSB code with the [GC coding style of SpiderMonkey](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/GC_Rooting_Guide).
+1. Provides a unified setup for both Cocos2d-x and Cocos.
+2. New __Cocos Launcher__ too! __Cocos Launcher__:
+  * one step creates a new project by either using our precompiled libraries or
+    source code.
+  * easily integrate SDKBOX for 3rd party services. Easily add Facebook, in-app
+    purchase and many more.
+  * provide feedback and get support, directly within __Cocos Launcher__, making
+    it simple to contact the engine team for help.
```
Comment (pandamicro) [inline docs/RELEASE_NOTES.md]: Provided feedback and support channel directly within **Cocos Launcher** for user to get help from the engine team easier.

**Comment by slackmoehrle** [comment]
Lexie send me notes to review and @pandamicro and I discussed this further via Beary chat. This is what we came up with if it helps:

##What's new in v3.10
1. Provides a unified setup for both Cocos2d-x and Cocos. This allows you to stay
   up to date with the latest Cocos2d-x releases. This includes using precompiled binaries
   and source code, in a single place. As always the source code is available
   from [GitHub]() but, there is no longer a `.zip` file available containing the
   source code. 
2. New **Cocos Launcher** tool! **Cocos Launcher** allows:
   - One step to create a new project with the engine, by using precompiled libraries or source code.
   - Easy integration with SDKBOX for 3rd party services: Facebook, IAP and many more.
   - Developers to provide feedback and get support, directly within **Cocos Launcher**, making
     it simple to contact the engine team for help.　　　　　
3. UI System:
   - Reimplemented `Scale9Sprite` to improve performance and reduce memory consumption.
   - Changed `PageView` to derive from `ListView`. `PageView` can add any widget as a child.
   - Added three new _overflow types_ to `Label`: **CLAMP**，**SHRINK**, **RESIZE_HEIGHT**.
   - Fixed a bug in `ClippingNode` that corrects its behavior when being set as a child.
4. New and improved Programmers Guide! Adds JavaScript, Cocos Studio, Cocos, bug fixes and more.
5. Improved **JavaScript Bindings**: follows [SpiderMonkey GC best practices](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/GC_Rooting_Guide) making it more robust.
6. Added `ccslog` to output Lua project log in Cocos output section.

**Pull request merged**